pygitguardian/models.py

```diff
@@ -341,6 +353,7 @@ def __init__(
         policy_break_count: int,
         policy_breaks: List[PolicyBreak],
         policies: List[str],
+        is_diff: bool,
```

Comment:
This is a breaking change: any code creating a ScanResult is going to fail until it's changed to pass a value for `is_diff`. You can avoid this with:

```suggestion
        is_diff: bool = False,
```